src/lib/include/xtgeo/point.hpp

```diff
@@ -0,0 +1,9 @@
+#ifndef POINT_H_
+#define POINT_H_
+
+struct Point
+{
+    double x, y, z;
+};
```

Comment:
Does this need `typedef Point`? It doesn't seem so, I cannot recall if C++ does this automatically

Reply:
Not needed for simple structures in C++